assets/css/v2/pre_fare/viewport.scss

```diff
@@ -1,23 +1,23 @@
 .screen-container,
-.simulation {
+#simulation {
   height: 1920px;
   overflow: hidden;
   // width of a single Pre-Fare display unit; "duo" installations are two of
   // these side-by-side
   --panel-width: 1080px;
 
-  &--duo {
+  &.simulation--duo {
```

Comment:
After we chatted, I realized this will now no longer work for the _normal_ screen page, because it results in a selector like `.screen-container.simulation--duo` (which will never match anything) instead of the intended `.screen-container--duo`. I think what we need to do is:

1. Keep these modifiers the same (e.g. `&--duo`)
2. Change line 2 to be something like `.simulation-viewport`
3. Change `pre_fare/simulation_screen_container.tsx` to _additionally_ apply this classname, with the modifier, to the element

The result should be:

```
<div id="simulation" class="simulation simulation-viewport simulation-viewport--duo">
```